cmd/rootMain.go

```diff
@@ -118,17 +119,20 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 		return exitCode{1}
 	}
 	srv := server.New(mountpath, socketpath, plugins, serverOpts)
-	if err := srv.Start(); err != nil {
+	successfullyLoadedPlugins, err := srv.Start()
+	if err != nil {
 		cmdutil.ErrPrintf("Unable to start server: %v\n", err)
 		return exitCode{1}
 	}
 	defer srv.Stop()
 
-	if plugin.IsInteractive() {
-		cmdutil.Println(`Welcome to Wash!
-  Wash includes several built-in commands: ls, wexec, find, meta, tail.
-  See commands run with wash via 'whistory', and logs with 'whistory <id>'.
-Try 'help'`)
+	if plugin.IsInteractive() && successfullyLoadedPlugins {
```

Comment:
Reason for the "successfullyLoadedPlugins" check is b/c it'd be weird for a first-time user to see a WARNING message being printed and then the "Welcome to Wash!" prompt showing up afterwards as if everything was successful.

I could also just omit the "Try 'ls'" part, and leave the Welcome message as-is.